x-pack/solutions/search/plugins/search_query_rules/server/lib/put_query_rules_ruleset_set.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { QueryRulesPutRulesetResponse } from '@elastic/elasticsearch/lib/api/types';
+import { ElasticsearchClient } from '@kbn/core/server';
+
+export const putRuleset = async (
```

Comment:
Please add a comment line to explain why this has defaults and should be updated before release.